Jenkinsfile.nightly-build

```diff
@@ -33,19 +33,19 @@ pipeline {
     }
     stages {
     	stage("Prepare workspace") {
-    		steps {
-    			library identifier: 'lisk-jenkins@nh-lisk-build', retriever: modernSCM(
-                 	[$class: 'GitSCMSource',
-                 	remote: 'https://github.com/LiskHQ/lisk-jenkins.git'])
-    		}
+            steps {
+            	library identifier: 'lisk-jenkins@master', retriever: modernSCM(
+                    [$class: 'GitSCMSource',
+                    remote: 'https://github.com/LiskHQ/lisk-jenkins.git'])
+            }
     	}
         stage('Build release') {
             steps {
             	script {
-					def release = liskBuild(branch: '1.0.0', network: 'dev')
-                 	env.VERSION =  release.version
-                 	env.RELEASE_FILE =  release.file
-                 	env.DESTINATION_FILE = "lisk-${env.VERSION}-${timestamp}-Linux-x86_64.tar.gz"
+                    def release = liskBuild(branch: "${env.BRANCH_NAME}", network: 'dev')
```

Comment:
@abelboldu Would it be `params.BRANCH_NAME` or `env.BRANCH_NAME`? Param is much better option so we can run this job with different parameters interactively.

Reply:
No, this is a nightly build,  it's automated and we cannot pass any parameter.

Reply:
Yes its nightly build job, but still having it parameterized makes more sense. Considering running nightly builds on two different branches for `main` net and `test` net separately.

Reply:
The approach is to run the job relative on the branch we are running. If we want to run on a different branch we can move the `Jenkinsfile` there

Reply:
@abelboldu Jenkins will not be aware that which branch belongs to which network. So we have to pass the network somehow to Jenkins job. In future that will be useful, when our build process can build any branch for any network.

Reply:
`env.BRANCH_NAME` is the branch Jenkins is currently working on, there is no need for a parameter.

Reply:
I just want not to hardcode the network. Network is based on branches right now, so not necessary that `master` branch is built with `dev` net.

Reply:
The *library* is being pulled from the `master` branch of lisk-jenkins; this has nothing to do with the `master` branch of `LiskHQ/lisk`